snapcraft/tests/unit/sources/test_errors.py

```diff
@@ -0,0 +1,47 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2018 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+from testtools.matchers import Equals
+
+from snapcraft.internal.sources import errors
+from snapcraft.tests import unit
+
+
+class ErrorFormattingTestCase(unit.TestCase):
```

Comment:
My idea was to keep all of these on the same file, `snapcraft/tests/unit/test_errors.py`, because that would make it very easy to compare the style, find inconsistencies and make improvements over all the messages.

The downside is that the file will be big. What do you think?

Reply:
No strong opinion personally - I did this as suggested in this PR earlier.

Reply:
I would really rather separate them and have them under a common module name `test_errors`

Reply:
I think keeping tests in roughly a layout that mirrors what they're testing makes sense. Testing `internal/x.py`? Then there should probably be a `snapcraft/tests/unit/test_x.py` Testing `internal/sources/errors.py`? Then a `snapcraft/tests/sources/test_errors.py` makes sense.

The one downside to this is that having one large file of error messages gives mpt something relatively easy to review.